doc/source/reference/arrays.indexing.rst

```diff
@@ -180,6 +180,16 @@ concepts to remember include:
    create an axis of length one. :const:`newaxis` is an alias for
    'None', and 'None' can be used in place of this with the same result.
 
+.. note::
+
+    NumPy slicing creates a :term:`view` instead of a copy as in the case of
+    oridinary Python objects. Care must be taken when extracting
```

Comment:
"builtin python sequences" would be better than "ordinary pyhon objects". Could add "like list and tuple" too.